### PR TITLE
MySql quick start

### PR DESCRIPTION
**Pull request by hh1680651**
Brief quick start doc for Twittermap with MySqlDB: 
https://github.com/ISG-ICS/cloudberry/wiki/Setup-TwitterMap-Using-Cloudberry-and-MySqlDB

### CODE REVIEW

File: examples/twittermap/script/dockerRunMySqlDB.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+echo "Starting docker-machine ..."
+docker-machine create --driver virtualbox default
+docker-machine start
+eval $(docker-machine env)
+
+HOST="$(echo -e "$(docker-machine ip)" | sed -e 's/^[[:space:]]*//')"
+PORT=3308
+
+echo "Editing config files ..."
```
Comment (JavierJia) [inline examples/twittermap/script/dockerRunMySqlDB.sh]: why do we need to edit the cloudberry config file in the script? 
I think the script should only take care of the setup the MySQL part. The config file is meant to be changed by a human.

File: examples/twittermap/script/dockerRunMySqlDB.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+#       CREATED: 08/14/2017 23:24:01 PM PDT
+#      REVISION:  ---
+#===============================================================================
+
+set -o nounset                      # Treat unset variables as an error
+
+echo "Starting docker-machine ..."
+docker-machine create --driver virtualbox default
+docker-machine start
+eval $(docker-machine env)
```
Comment (JavierJia) [inline examples/twittermap/script/dockerRunMySqlDB.sh]: why do we need to start the docker-machine? it seems unnecessary.

Comment (hh1680651) [inline examples/twittermap/script/dockerRunMySqlDB.sh]: Thanks! I've checked and removed docker-machine, using localhost's url instead.

File: examples/twittermap/script/dockerRunMySqlDB.sh (hunk 1 above)
Comment (hh1680651) [inline examples/twittermap/script/dockerRunMySqlDB.sh]: So.. Should I write these 3 command lines into set up step's instead?

**Comment by JavierJia** [comment]
Yes.

**Pull request merged**